misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/natives/mock/ObjectMock.java

```diff
@@ -17,6 +17,7 @@
  */
```

Comment:
There are some unused class definitions in this class.
Shall we remove them if we don't use them anywhere?
eg: `validateFunctionName` in L118, `validateFieldName` in L139

Reply:
`validateFunctionName` and `validateFieldName` are used as part of the interop in `mock.bal`